rpmlint/checks/MenuXDGCheck.py

```diff
@@ -77,7 +59,43 @@ def check_file(self, pkg, filename):
                     error_printed = True
             if not error_printed:
                 self.output.add_info('E', pkg, 'invalid-desktopfile', filename)
-        if not is_utf8(f):
-            self.output.add_info('E', pkg, 'non-utf8-desktopfile', filename)
 
         self.parse_desktop_file(pkg, root, f, filename)
+
+    def _handle_parser_error(self, pkg, filename, e):
+        """
+        Determine what to do with a caught configparser error.
+        """
+        # I would love to use switch, however, each warning is printed differently
+        if (isinstance(e, cfgparser.MissingSectionHeaderError)):
+            self.output.add_info('E', pkg, 'desktopfile-missing-header', filename)
+        elif (isinstance(e, cfgparser.DuplicateSectionError)):
+            self.output.add_info('E', pkg, 'desktopfile-duplicate-section', filename,
+                                 '[%s]' % e.section)
+        elif (isinstance(e, cfgparser.DuplicateOptionError)):
+            self.output.add_info('E', pkg, 'desktopfile-duplicate-option', filename,
+                                 '[%s]/%s' % (e.section, e.option))
```

Comment:
please use here and on other above/bellow the normal python formated string: `f'broken stuff {variable}'`